tests/foreman/api/test_subscription.py

```diff
@@ -181,3 +189,56 @@ def test_positive_delete_manifest_as_another_user(self):
             data={'organization_id': org.id}
         )
         self.assertEquals(0, len(Subscription.list({'organization-id': org.id})))
+
+    @tier2
+    def test_positive_subscription_status_disabled(self):
+        """Verify that Content host Subscription status is set to 'Disabled'
+         for a golden ticket manifest
+
+        :id: d7d7e20a-e386-43d5-9619-da933aa06694
+
+        :expectedresults: subscription status is 'Disabled'
+
+        :BZ: 1789924
+
+        :CaseImportance: Medium
+        """
+        org = entities.Organization().create()
+        with manifests.clone(name='golden_ticket') as manifest:
+            upload_manifest(org.id, manifest.content)
+        rh_repo_id = enable_rhrepo_and_fetchid(
+            basearch='x86_64',
+            org_id=org.id,
+            product=PRDS['rhel'],
+            repo=REPOS['rhst7']['name'],
+            reposet=REPOSET['rhst7'],
+            releasever=None,
+        )
+        rh_repo = entities.Repository(id=rh_repo_id).read()
+        rh_repo.sync()
+        custom_repo = entities.Repository(
+            product=entities.Product(organization=org).create(),
+        ).create()
+        custom_repo.sync()
+        lce = entities.LifecycleEnvironment(organization=org).create()
+        cv = entities.ContentView(
+            organization=org, repository=[rh_repo_id, custom_repo.id],
+        ).create()
+        cv.publish()
+        cvv = cv.read().version[0].read()
+        promote(cvv, lce.id)
```

Comment:
You can drastically speed up your test if you simply loose the whole L223-L229 part. The test is not focused on lces nor cvs, so if you simply went with the default: Library env and the default CV, it would have no effect on functionality, but a huge effect on time efficiency :)